### DISCUSSION TITLE
move items form organisation

### DISCUSSION THREAD

**Discussion by sadden3194** [dani-garcia/vaultwarden#3560] [General]
Hello everyone, 

I am the admin of a vaultwarden organisation.
This organisation is to be deleted, but the records are to be transferred to a vault of my own.

If I understand correctly:
I cannot export organisation records and I cannot move those records from there to my vault.

Only way:
Duplicate these records - but each one separately?

Is there no way to select them all and duplicate them all in one step?

**Comment by BlackDex** [comment]
If you are an admin you can export all items, and later import them in your personal vault. Not sure why you think you can't though?

There is indeed no select all and clone to personal.

**Comment by sadden3194** [comment]
![Bildschirmfoto 2023-06-02 um 13 35 02](https://github.com/dani-garcia/vaultwarden/assets/17011498/6cc5d6de-98bf-46bb-9c1a-94115e810dbf)

??

**Reply by BlackDex** [reply]
You need to go to Organizations.
https://bitwarden.com/help/export-your-data/#export-an-organization-vault

**Reply by sadden3194** [reply]
oh god!
thanks!
